SConstruct

```diff
@@ -829,28 +829,7 @@ if env["ARNOLD_ROOT"] :
 # Definitions for the libraries we wish to build
 ###############################################################################################
 
-vTuneRoot = env.subst("$VTUNE_ROOT")
-
-if env["PLATFORM"] == "win32" : 
-	gafferLib = {
-		"envAppends" : {
-			"LIBS" : [ "Advapi32" ]
-		}
-	}
-else:
-	gafferLib = {}
-
-if os.path.exists( vTuneRoot ):
-	gafferLib = {
-		"envAppends" : {
-			"CXXFLAGS" : [ systemIncludeArgument, "$VTUNE_ROOT/include", "-DGAFFER_VTUNE"],
-			"LIBPATH" : [ "$VTUNE_ROOT/lib64" ],
-			"LIBS" : [ "ittnotify" ] + gafferLib.get( "envAppends", {} ).get( "LIBS", [] )
-		},
-		"pythonEnvAppends" : {
-			"CXXFLAGS" : [ "-DGAFFER_VTUNE"]
-		}
-	}
+gafferLib = {}
```

Comment:
We don't need this variable any more - we can just put `"Gaffer" : {}` inline in the `libraries` definition.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4524/commits/718aab85f3f270369b97f92bfd14f0e70852db1d